hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/SelfRenewingLease.java

```diff
@@ -71,7 +72,7 @@
 
   public static class LeaseException extends AzureBlobFileSystemException {
     public LeaseException(Throwable t) {
-      super(ERR_ACQUIRING_LEASE + ": " + t.getMessage());
+      super(ERR_ACQUIRING_LEASE, t);
```

Comment:
I think I'd keep that t text in the superclass text, in case a deep tree causes the nested cause not to be listed.

but: use toString() (implicitly) rather than getMessage, because some exceptions (NPW) have a null message.